exporter/datadogexporter/config.go

```diff
@@ -315,6 +308,16 @@ type LogsConfig struct {
 
 	// DumpPayloads report whether payloads should be dumped when logging level is debug.
 	DumpPayloads bool `mapstructure:"dump_payloads"`
```

Comment:
Does this config apply to logs-agent? If not, please document

Reply:
Added this to the config validate function